docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/customizing-policy.adoc

```diff
@@ -0,0 +1,30 @@
+= Customizing Policy
+
+An Enterprise Contract Policy defines which checks are performed by EC during integration tests and
+release. If EC reports violations, the best course of action, in most cases, is to adjust the build
```

Comment:
Users can face issues if the policies used are out of sync. Should we recommend that ITS policies are at least as strict as the RPA policies?

Reply:
That's a good idea but I think it's outside the scope of this PR. Maybe that belongs in an ADR?